README.md

```diff
@@ -177,14 +177,14 @@ A chai [should](https://www.chaijs.com/api/bdd/) instance, containing the `bignu
 
 ---
 
-### shouldFail
-Collection of assertions for failures (similar to [chai's `throw`](https://www.chaijs.com/api/bdd/#method_throw)). `shouldFail` will accept any exception type, but more specific functions exist and their usage is encouraged.
+### expectFailure
+Collection of assertions for failures (similar to [chai's `throw`](https://www.chaijs.com/api/bdd/#method_throw)). `expectFailure` will accept any exception type, but more specific functions exist and their usage is encouraged.
 
-#### async shouldFail.reverting (promise)
+#### async expectFailure.revert (promise)
 Only accepts failures caused due to an EVM revert (e.g. a failed `require`).
 
-#### async shouldFail.reverting.withMessage (promise, message)
-Like `shouldFail.reverting`, this helper only accepts failures caused due to an EVM revert (e.g. a failed `require`). Furthermore, it checks whether revert reason string includes passed `message`. For example:
+#### async expectFailure.revert.withMessage (promise, message)
```

Comment:
I was thinking if it would be a good idea to merge `revert` and `revert.withMessage`, by having a single function with an optional `message` argument. But making `message` a mandatory argument feels safer because we don't want people to forget to test for a specific revert reason. We know that not doing so can lead to bad tests.

But then, if that's our guiding principle, we should make `revert.withMessage` the easiest/least friction option. So I would make `expectFailure.revert`/`expectRevert` take a mandatory `message` argument, and have a separate function or a separate way of saying that you don't expect a specific reason. Perhaps `expectRevert.unspecified(promise)`?

Reply:
There's an issue for this (#30), but I hadn't considered the possibility of making the argument mandatory. I worry that it may be annoying to our users though. But a good error message might help.

```
No revert reason specified: call expectRevert with the reason string, or use expectRevert.unspecified if your 'require' statement doesn't have one.
```

Reply:
Yes! Useful error messages :raised_hands: :rainbow: